Dockerfile.skaffold

```diff
@@ -0,0 +1,27 @@
+ARG GO_VERSION=1.19.9
+ARG crossbuild_image="docker.elastic.co/beats-dev/golang-crossbuild"
+ARG AGENT_VERSION=8.9.0-SNAPSHOT
```

Comment:
we need to update this with each version bump. 
do you think we can read a file and inject it here?